test/test-framework/deploy/olm-catalog/memcached-operator/memcached-operator.package.yaml

```diff
@@ -0,0 +1,7 @@
+packageName: memcached-operator
```

Comment:
It will not work. The pkgName is used to build the name of the Application in the registry. So, it cannot be the same name used to push the image. 

It would fail in the test `CVP/operator-packagename-uniqueness` . See here an example: https://s3.amazonaws.com/cvp-co-pipeline-artifacts/cvp-community-operator-metadata-validation-test/679/269d220e-eeb1-43ba-afb1-1306edcaa5f4/operator-packagename-uniqueness-output.txt

Reply:
@camilamacedo86 I could be wrong (@estroz can correct me, if so), but these test files are used in isolation and the tests don't involve the published operatorhub registry or the CVP, so there should be no conflict issue with this.

Reply:
So, /lgtm for me. I'd suggest replacing anyway for something as `memcached-operator-application` in the case we decide to use as an example in any other place.

Reply:
Yeah the package name does not matter here since this data is purely internal.